<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>igwLineParametersWidget</class>
 <widget class="QWidget" name="igwLineParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>线参数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showLineCheckBox">
         <property name="text">
          <string>显示线</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Point1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Point1XEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="Point1YEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="Point1ZEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Point2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Point2XEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Point2YEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Point2ZEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="XAxisButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>X Axis</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="YAxisButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Y Axis</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ZAxisButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Z Axis</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>中心线</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
